source/simulator/assemblers/newton_stokes.cc

```diff
@@ -109,16 +109,37 @@ namespace aspect
                 for (unsigned int j = 0; j < stokes_dofs_per_cell; ++j)
                   if (scratch.dof_component_indices[i] ==
                       scratch.dof_component_indices[j])
-                    data.local_matrix(i, j) += ((2.0 * eta * (scratch.grads_phi_u[i] * scratch.grads_phi_u[j]))
-                                                + one_over_eta * this->get_pressure_scaling() * this->get_pressure_scaling()
-                                                * (scratch.phi_p[i] * scratch.phi_p[j]))
+                    data.local_matrix(i, j) += (
+                                                 // top left block: for the current case with
+                                                 // derivative_scaling_factor==0 the top left block
+                                                 // of the system matrix only contains the usual
+                                                 // Stokes term. so approximate this block in the
```

Comment:
so->So

Reply:
Sure. Though we have many comments that start sentences with lower-case letters. For example, the first line of this comment starts with a lower-case letter. 

But happy to change -- new version now online.

Reply:
ah, I didn't notice that, I now see it everywhere... Anyway, not a big issue for me. Will merge it when the testers are done :)